packages/monitor-v2/src/monitor-og/common.ts

```diff
@@ -206,6 +207,9 @@ export const initMonitoringParams = async (env: NodeJS.ProcessEnv, _provider?: P
     signer = await getSigner(env, provider);
   }
 
+  // By default, disputes on server errors will be submitted if there is less than 1 hour left in the dispute period.
+  const serverErrorDisputePeriod = env.SERVER_ERROR_DISPUTE_PERIOD ? Number(env.SERVER_ERROR_DISPUTE_PERIOD) : 3600;
```

Comment:
Maybe checking that `isNaN(serverErrorDisputePeriod) == false` and throw otherwise could be safer